spec/poros/quotes_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe Quotes do 
+  it 'instantiates' do
+    @quotes_array = QuotableFacade.quote_search("I am")
+
+    @quotes_array.each do |quote|
+      expect(quote).to be_a(Quotes)
+    end
```

Comment:
This definitely works, but there are a couple things I don't love about the approach are using the facade to get the data to send to the poro. 1) If your facade ever changes (ex: the name of the method), then this test breaks even though it's unrelated. 2) It's making an API call when all we really need to test this is some sample data in the format that the `QuotePoro` class is expecting. We don't need to use VCR or other mocking tools to do this. 3) If the first result from the API ever changes, this test fails. We can just make up data ourselves and do something like this instead:
```ruby
  before :each do
    @quote = Quotes.new({ content: "If I am not for myself, who will be for me? If I am not for others, what am I? And if not now, when?",
                         author: "Rabbi Hillel",
                         tags: ["famous-quotes", "wisdom"] })
  end
```
Then you can test the actual stored attribute values for your `Quotes` object as well, instead of having to do `String` or `Array` which isn't very specific.